File: Python/Day 1/Day1Part1.py
f = open("/Users/mcneillc/Documents/advent-of-code-2018/Python/Day 1/Day1input.txt","r")
frequency_changes = f.readlines()
f.close()

# current_frequency = 0
# for change in frequency_changes:
#     current_frequency += int(change)
# print(current_frequency)

frequency_changes = [int(i) for i in frequency_changes]

total = sum(frequency_changes)

print(total)

File: Scala/src/Day1/Day1Part1.scala
package Day1

import scala.io.Source

object Day1Part1 {

  def frequency_changes(): Int = {

    val f = Source.fromFile("/Users/mcneillc/Documents/advent-of-code-2018/Scala/src/Day1/Day1input.txt").getLines

    f.toList.map(_.toInt).sum

  }

  def main(args: Array[String]): Unit = {

    println(frequency_changes())

  }

}
